httpclient-parent/httpclient/src/main/java/io/yupiik/uship/httpclient/core/listener/impl/HARDumperListener.java

```diff
@@ -85,4 +83,10 @@ protected void onEntry(final Har.Entry entry) {
 
     public static class Har extends BaseHARDumperListener.Har { // backward compat
     }
+
+    public static class Configuration extends BaseConfiguration {
+        protected Configuration(final Path output, final Clock clock, final Logger logger) {
```

Comment:
should be public, no?

Reply:
Yes, and i think the same for NDJSONDumperListener.Configuration, constructor should be public?

Reply:
Yep, got fixed yesterday